gcloud/storage/blob.py

```diff
@@ -358,7 +402,8 @@ def _check_response_error(request, http_response):
             raise make_exception(faux_response, http_response.content,
                                  error_info=request.url)
 
-    def upload_from_file(self, file_obj, rewind=False, size=None,
+    # pylint: disable=too-many-locals
```

Comment:
There is still no closing `enable=too-many-locals` later on.

Which method has too many locals?